src/__tests__/commands/flex/plugins/start.test.ts

```diff
@@ -2,21 +2,24 @@ import { expect, createTest } from '../../../framework';
 import FlexPluginsStart from '../../../../commands/flex/plugins/start';
 
 describe('Commands/FlexPluginsStart', () => {
-  const { sinon, start } = createTest(FlexPluginsStart);
+  const { sinon, start: _start } = createTest(FlexPluginsStart);
 
   afterEach(() => {
     sinon.restore();
   });
 
-  start()
-    .setup((instance) => {
-      sinon.stub(instance, 'runScript').returnsThis();
-    })
-    .test(async (instance) => {
-      await instance.doRun();
+  const start = (args?: string[]) =>
```

Comment:
This is too much refactoring that removes from the clarity of what the test is doing. It's fine to repeat the setup part of the test.